rotkehlchen/chain/avalanche/manager.py

```diff
@@ -0,0 +1,229 @@
+import logging
+from typing import Any, Dict, List, Optional
+from datetime import datetime
+
+from web3 import HTTPProvider, Web3
+from web3.datastructures import MutableAttributeDict
+from web3.exceptions import BadFunctionCallOutput
+
+from rotkehlchen.chain.avalanche.graph import Graph
+from rotkehlchen.errors import (
+    BlockchainQueryError,
+    DeserializationError,
+    RemoteError,
+)
+from rotkehlchen.externalapis.covalent import Covalent
+from rotkehlchen.fval import FVal
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.typing import ChecksumEthAddress, Timestamp
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.misc import from_wei, hex_or_bytes_to_str
+
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+DEFAULT_ETH_RPC_TIMEOUT = 10
```

Comment:
If this is the same timeout used with ETH RPC then why not move this into a common constants file under `/chain/constants.py` and use it from there? And probably change its name to something else as it's not longer ETH specific timeout but also avalanche (and probably in the future more EVM based chains)